types/three/src/renderers/shaders/UniformsUtils.d.ts

```diff
@@ -1,7 +1,8 @@
 import { UniformsGroup } from '../../core/UniformsGroup.js';
+import { IUniform } from './UniformsLib.js';
 
-export function cloneUniforms(uniforms_src: any): any;
-export function mergeUniforms(uniforms: any): any;
+export function cloneUniforms(uniformsSrc: { [uniform: string]: IUniform }): { [uniform: string]: IUniform };
+export function mergeUniforms(uniforms: { [uniform: string]: IUniform }): { [uniform: string]: IUniform };
```

Comment:
I this should take an array of objects, right?

Reply:
Oh, you're right, good catch :+1: